null name

Name ``null`` in my opinion is obtained if an ``unreadable character`` is used in the name of the ``original name``

java -version
```
openjdk version "19.0.2" 2023-01-17
OpenJDK Runtime Environment (build 19.0.2+7)
OpenJDK 64-Bit Server VM (build 19.0.2+7, mixed mode)
```

launched by my project
https://github.com/publicLibs/continuous-Jazzer

target:
oneclick layout plugin
https://yed.yworks.com/support/manual/layout_oneclick.html

```
FUZZER ARG[--keep_going=2000]
FUZZER ARG[-detect_leaks=1]
FUZZER ARG[-max_total_time=300]
FUZZER ARG[--autofuzz_ignore=java.lang.NullPointerException]
FUZZER ARG[--autofuzz=y.G.F.C.U$1::ā(y.G.F.C.G,y.G.F.C.G,y.G.F.G)]
FUZZER ARG[--instrumentation_excludes=com.fasterxml.jackson.**:org.yaml.snakeyaml.**]
```

```
== Java Exception: java.lang.NoClassDefFoundError: y/G/t
        at y.G.F.C.U$1.ā(Unknown Source)
        at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
        at java.base/java.lang.reflect.Method.invoke(Method.java:578)
Caused by: java.lang.ClassNotFoundException: y.G.t
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
```
```java
public class Crash_92951f0318fedf71a82236f4fca8f9f97570e38c {
  public static void main(String[] args) throws Throwable {
    Crash_92951f0318fedf71a82236f4fca8f9f97570e38c.class.getClassLoader().setDefaultAssertionStatus(true);
    (new null((y.G.F.C.U) null)).ā(new y.G.F.C.G((y.G.F.C.D) null), (y.G.F.C.G) null, (y.G.F.G) null);
  }
}
```

--- a/src/main/java/com/code_intelligence/jazzer/autofuzz/Meta.java
+++ b/src/main/java/com/code_intelligence/jazzer/autofuzz/Meta.java
@@ -601,6 +601,9 @@ public class Meta {
           implementingClasses = children.getStandardClasses()
                                     .filter(info -> !Modifier.isAbstract(info.getModifiers()))
                                     .filter(info -> lookup.isAccessible(info, info.getModifiers()))
+                                    // Filter out anonymous and local classes, which can't be
+                                    // instantiated in reproducers.
+                                    .filter(info -> info.getName() != null)
                                     .loadClasses();
           implementingClassesCache.put(type, implementingClasses);
         }

